### ISSUE TITLE
About 300 missing Jest tests

### ISSUE THREAD

**Issue by justingrant** [fullcalendar/temporal#2]
Love this polyfill!  Really impressive work.

Following up from https://github.com/tc39/proposal-temporal/issues/2060#issuecomment-1041141522, this polyfill seems to be missing about 300 Jest tests that were originally in https://github.com/tc39/proposal-temporal but have since been migrated to Test262. PlainDate and PlainTime tests are missing about 100 tests each, with smaller numbers missing for other test files. 

You'll be able to find the missing Jest tests here: https://github.com/js-temporal/temporal-polyfill/tree/main/test.  (We forked proposal-temporal into temporal-polyfill before we started migrating non-Test262 tests to Test262.)

temporal-polyfill has almost no changes from proposal-temporal's tests, so it should be easy to diff your test files with temporal-polyfill to find the missing tests. (Running prettier first makes this easier by removing trivial differences like semicolons.)

Another alternative to get coverage for the missing tests is just to run Test262, but while working on https://github.com/js-temporal/temporal-polyfill we've found it convenient for development to have tests that run in 10 seconds as opposed to 10 minutes for Test262, even if they overlap.

There are also unique tests in Test262 that have never been in Jest tests in any repo, so you'll probably want to run Test262 against this polyfill too. I'll file a separate issue for that.

Here's a rough comparison of test counts in this repo vs. temporal-polyfill, although I guessed at the right regex to catch all tests so YMMV. I did manually validate that ~100 tests were missing from PlainDate so I have at least some trust in these numbers.

<img width="844" alt="image" src="https://user-images.githubusercontent.com/277214/154415776-9cd3d091-4bcf-4cd0-838a-f9f295d12253.png">

**Comment by arshaw** [comment]
When I ported over the tests from the proposal-temporal repo to this @fullcalendar/temporal a little while back, I did it from [this commit](f0bc1c5a086e954e29fe1ccce0fc33383b4eb4d2) (Sept 2021) and have not updated them since.

My plan is to continue to get those tests from the past to pass. The big TODO is to get user-calendar/timezone tests to pass, the tests that involve subclassing the built-in objects.

After I do that, I'll port over the missing tests by re-importing all tests from https://github.com/js-temporal/temporal-polyfill/tree/main/test

**Comment by justingrant** [comment]
> I've spot-checked it on a few non-ISO calendars with positive results

I'll be interested to see how it handles the Chinese calendar which has leap months (like Hebrew) so almost certainly will need some custom code.

**Comment by arshaw** [comment]
Hi @justingrant, I re-copied over all [tests from the @js-temporal/polyfill repo](https://github.com/js-temporal/temporal-polyfill/tree/main/test) and got them to pass. They are passing in Node 17, [Node 16](https://github.com/fullcalendar/temporal/runs/5857411785?check_suite_focus=true#step:8:28), and Node 14.

I still need to address [some of your other feedback](https://github.com/fullcalendar/temporal/issues/6). Thank you for your guidance.
